assets/js/app.js

```diff
@@ -467,4 +468,274 @@ function setupFormsWithPatchMethod() {
         }
     });
   });
+}
+
+const metricsCharts = [];
+const colorPalette = [
+  {
+    color: '#5470c6',
+    class: 'blue-600'
+  },
+  {
+    color: '#91cc75',
+    class: 'green-400'
+  },
+  {
+    color: '#fac858',
+    class: 'amber-400'
+  },
+  {
+    color: '#ee6666',
+    class: 'red-400'
+  },
+  {
+    color: '#73c0de',
+    class: 'sky-300'
+  },
+  {
+    color: '#3ba272',
+    class: 'emerald-600'
+  },
+  {
+    color: '#fc8452',
+    class: 'orange-500',
+  }
+];
+
+function setupMetricsCharts() {
+  const metricsContainer = document.querySelector('#metrics-container');
+  if (!metricsContainer) {
+    return;
+  }
+
+  $(document).ready(function () {
```

Comment:
We already call `setupMetricsCharts` in `$(function () {})` which is shorthand for `$(document).ready(function () {})`, so I think we don't need it call it again here.

Reply:
fixed